src/backend/src/controllers/design-reviews.controllers.ts

```diff
@@ -116,4 +116,18 @@ export default class DesignReviewsController {
       next(error);
     }
   }
+
+  // Mark the current user as confirmed for the given design review
+  static async markUserConfirmed(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { availability } = req.body;
```

Comment:
you named it meetingTimes in the route, that's why you're getting null (should be availability)